### ISSUE TITLE
F06 Mapping: Fixed Mapping for AWARD_CONTRACT with NO_AWARDED_CONTRACT

### ISSUE THREAD

**Issue by fokoug** [OP-TED/ted-rdf-mapping#211]
![ContractInvalid184749-2021](https://user-images.githubusercontent.com/24571803/194187246-a84bfe25-4f0f-4459-adc9-2d3376c132ec.png)
These invalid are du to an activation of the AWARD_CONTRACT Xpath mapping :
![contractInvalidExplain184749_2021](https://user-images.githubusercontent.com/24571803/194187302-95f7d39e-d7d0-43cf-8856-6942dfc070aa.png)
But in the XML file, in the AWARD_CONTRACT tag we don't have any others informations: 
![contractXMLInvalid184749-2021](https://user-images.githubusercontent.com/24571803/194187393-eae9f199-dd44-4543-be03-e7e1094ca019.png)

**Comment by csnyulas** [comment]
This is not a real issue. In fact in the conceptual mapping we have a Field XPath Condition, that says that this mapping should be created on the `AWARD_CONTRACT` XPath only if it has a sub-element called `AWARDED_CONTRACT`. Since in the case of notice `184749-2021`, where this issue occured, we don't have any awarded lot (in the screenshot provided in the issue detail one can see that both Lot's are non-awarded), we do not need to create a Contract instance (i.e. a notice will not announce any contracts).

![image](https://user-images.githubusercontent.com/2441046/194388110-f4ad76df-33f5-46bc-b7eb-4d1132a2dc3b.png)

Basically, this is a false negative, or fake "invalid" result status. We should consider improving the validator to avoid creating such fake invalids, but it is not trivial.
